### PR TITLE
templates: Node.js event template to return event

### PR DESCRIPTION
**Pull request by lance**
This commit modifies the Node.js template so that it returns a CloudEvent. The
tests are also modified to test for CloudEvent attributes and headers. Additionally
the faas-js-runtime bump reverses the parameters for the function.

Fixes: #190
Fixes: #194

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

**Comment by lance** [comment]
Converting to draft so that I can also include a fix for https://github.com/boson-project/faas/issues/194

**Comment by lance** [comment]
@boson-project/core PTAL

File: templates/node/events/README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
 # Node.js Cloud Events Function
 
-Welcome to your new Node.js function project! The boilerplate function code can be found in [`index.js`](./index.js). This function is meant to respond exclusively to [Cloud Events](https://cloudevents.io/), but you can remove the check for this in the function and it will respond just fine to plain vanilla incoming HTTP requests. Additionally, this example function is written asynchronously, returning a `Promise`. If your function does not perform any asynchronous execution, you can safely remove the `async` keyword from the function, and return raw values intead of a `Promise`.
+Welcome to your new Node.js function project! The boilerplate function code can
```
Comment (zroubalik) [inline templates/node/events/README.md:3]: Just a nit and could be solved later, do we want to use `Function` or `function`? I have seen both version across the files, we should probably settle on some convention (I am inclined to use capital `F`)

**Review by zroubalik — APPROVED** [review]
LGTM

File: templates/node/events/README.md (hunk 1 above)
Comment (lance) [reply templates/node/events/README.md:3]: Hahah! https://github.com/boson-project/faas/pull/206#pullrequestreview-524286346

As the CLI usability guru, Roland recommends lowercase. ;)

Comment (zroubalik) [reply templates/node/events/README.md:3]: OK 😄

**Pull request merged**